### DISCUSSION TITLE
cache dir and data dirs

### DISCUSSION THREAD

**Discussion by stanly0726** [ayamir/nvimdots#558] [Q&A] [answered]
in this part of lua/core/init.lua

https://github.com/ayamir/nvimdots/blob/70d2a5e68899955e79001d4ad86cf5057b19b608/lua/core/init.lua#L3-L21

i don't quite under stand what these folder are for and can't find references to these path in whole repo
can someone tell me what this is for?

**Comment by ayamir** [comment, marked as answer]
Only `undo` directory is used in fact. Others are ready for users to utilize them like you can store `gtags` index files to `~/.cache/tags`. Of course, you can delete all of them except `undo` dir.

**Reply by stanly0726** [reply]
got it , 
also i cant quite wrap my head around when to use map_cr and map_cmd in terms of key binding
any rule of thumb for that?

**Reply by ayamir** [reply]
https://github.com/ayamir/nvimdots/wiki/Usage
